### DISCUSSION TITLE
DDC/CI compatible USB adapters

### DISCUSSION THREAD

**Discussion by emoacht** [emoacht/Monitorian#117] [Compatibility]
According to the [article](https://support.displaylink.com/knowledgebase/articles/1096210-usb-monitor-brightness-and-contrast-control-via-sc) in DisplayLink, its chips support DDC/CI.

> V9.0 Windows software enables a user to change brightness and contrast on DisplayLink devices using any utility complying with the DDC/CI standard.

The products which use its chips are shown in this [list](https://www.displaylink.com/products/usb-adapters). I welcome information on their actual compatibilities with Monitorian in this discussion.

**Comment by Adelscott** [comment]
Saddly it doesn't work with "Cable Matters USB 3.0 to DisplayPort Adapter". Tried with default windows drivers and with latest DisplayLink drivers (23 Sep 2020)

**Reply by emoacht** [reply]
Thank you for the report.
I expected an adaptor with DisplayLink would support DDC/CI but in reality, it was not true.
Too bad.
